build_web_compilers/web/stack_trace_mapper.dart

```diff
@@ -81,6 +81,7 @@ class LazyMapping extends Mapping {
 
   LazyMapping(this._provider);
 
+  // ignore: unreachable_from_main
```

Comment:
Looks like a new warning, was't sure this API is used at runtime somewhere so I hesitated to remove it:)

Reply:
Ignoring it is the correct thing to do for now. It looks to me like a bug in the lint.

https://github.com/dart-lang/linter/issues/4495